# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: gfx/layers/ipc/PWebRenderBridge.ipdl
# Commit: 77ef8e8bc669
# Full Hash: 77ef8e8bc669c9ecab4d94505b11b3cb732446c5
# Author: Alexis Beingessner <a.beingessner@gmail.com>
# Date: 2019-07-15 21:39:46
# Regressor Bug: 1547351
# File Overlap Count: 2
# Description:
#   Bug 1547351 - Automatically set content process render root r=kats
#   
#   This implements the idea of automatically setting a content proc's
#   render root based on the render root enclosing the iframe that
#   points to it. There was a bit of cleanup in here that was a bit
# ==============================================================================

diff -r b594f8e47025 -r 77ef8e8bc669 gfx/layers/ipc/PWebRenderBridge.ipdl
--- a/gfx/layers/ipc/PWebRenderBridge.ipdl	Mon Jul 15 13:20:31 2019 +0000
+++ b/gfx/layers/ipc/PWebRenderBridge.ipdl	Mon Jul 15 13:18:44 2019 +0000
@@ -49,14 +49,13 @@
   async DeleteCompositorAnimations(uint64_t[] aIds);
   async SetDisplayList(RenderRootDisplayListData[] displayLists,
                        OpDestroy[] toDestroy, uint64_t fwdTransactionId, TransactionId transactionId,
-                       IdNamespace aIdNamespace, bool containsSVGGroup,
+                       bool containsSVGGroup,
                        VsyncId vsyncId, TimeStamp vsyncStartTime,
                        TimeStamp refreshStartTime, TimeStamp txnStartTime, nsCString txnURL, TimeStamp fwdTime,
                        CompositionPayload[] payloads);
-  async EmptyTransaction(FocusTarget focusTarget, uint32_t aPaintSequenceNumber,
+  async EmptyTransaction(FocusTarget focusTarget,
                          RenderRootUpdates[] renderRootUpdates,
                          OpDestroy[] toDestroy, uint64_t fwdTransactionId, TransactionId transactionId,
-                         IdNamespace aIdNamespace,
                          VsyncId vsyncId, TimeStamp vsyncStartTime,
                          TimeStamp refreshStartTime, TimeStamp txnStartTime,
                          nsCString txnURL, TimeStamp fwdTime,